lms/templates/problem.html

```diff
@@ -28,12 +28,13 @@ <h3 class="hd hd-2 problem-header" id="${ short_id }-problem-title" aria-describ
 
     <div class="submit-attempt-container">
       <button type="button" class="submit btn-brand" data-submitting="${ submit_button_submitting }" data-value="${ submit_button }" data-should-enable-submit-button="${ should_enable_submit_button }" aria-describedby="submission_feedback_${short_id}" ${'' if should_enable_submit_button else 'disabled'}>
-          <span class="submit-label" aria-hidden="true">${ submit_button }</span><span class="sr">${_("Submit your answer. Some problems have options such as save, reset, hints, or show answer. These options follow the Submit button.")}</span>
+          <span class="submit-label" aria-hidden="true">${ submit_button }</span><span class="sr">${_("Submit your answer")}</span>
       </button>
       <div class="submission-feedback" id="submission_feedback_${short_id}">
-      % if attempts_allowed:
+        % if attempts_allowed:
           ${ungettext("You have used {num_used} of {num_total} attempt", "You have used {num_used} of {num_total} attempts", attempts_allowed).format(num_used=attempts_used, num_total=attempts_allowed)}
-      % endif
+        % endif
+        <span class="sr">${_("Some problems have options such as save, reset, hints, or show answer. These options follow the Submit button.")}</span>
```

Comment:
Good point @cahrens.  I like to include the space to ensure that it is always read properly.  However, in this case, the two lines of text are on their own lines in the DOM, so a space should be implied.  I tested on multiple platforms and screen readers and they all imply a space.  If both spans were on the same line with no spaces in between them, all screen readers would concatenate the two strings with no space between.